src/shogun2-core/shogun2-dao/src/main/java/de/terrestris/shogun2/dao/AbstractTokenDao.java

```diff
@@ -0,0 +1,27 @@
+package de.terrestris.shogun2.dao;
+
+import org.springframework.stereotype.Repository;
+
+import de.terrestris.shogun2.model.token.Token;
+
+/**
+ * As the {@link Token} class is abstract, this class will also be abstract.
+ * There will also be NO {@link Repository} annotation here.
```

Comment:
So the import is obnly ther to link to the not-used annotation? Can't we simpy use the text `org.springframework.stereotype.Repository`?